app/src/main/java/com/android/consumerfinancehalan/utils/ApiResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.android.consumerfinancehalan.utils
+
+sealed class ApiResponse<out T>{
```

Comment:
why ApiResponse added to utils package ?

Reply:
this package is wrong?